modules: mcuboot: Allow multi-image to be used without BOOT_UPGRADE_ONLY (#8699)
nrfconnect/sdk-nrf: doc/nrf/ug_nrf5340.rst

* :kconfig:option:`CONFIG_PCD_APP` - Enable commands exchange with the network core.
* :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple update partitions by setting this option to ``2``.

.. note::

>    The application core can be reverted however this will brick the network core upon revert as it will fill the network core with whatever is currently in the RAM that pcd uses.

Reviewer: ```suggestion
   The application core can be reverted, but doing so bricks the network core upon revertion, as the reversion process fills the network core with the content currently in the RAM that pcd uses.
```

>    To enable this the Kconfig ``USE_NRF53_MULTI_IMAGE_WITHOUT_UPGRADE_ONLY`` needs to be defined in the project level KConfig.

Reviewer: ```suggestion
   To enable this, define the ``USE_NRF53_MULTI_IMAGE_WITHOUT_UPGRADE_ONLY`` kconfig option in the project-level KConfig file.
```

>    When this is option is defined it can be enabled setting :kconfig:option`CONFIG_USE_NRF53_MULTI_IMAGE_WITHOUT_UPGRADE_ONLY`.

Reviewer: ```suggestion
   When this is option is defined, you can enable it by setting :kconfig:option`CONFIG_USE_NRF53_MULTI_IMAGE_WITHOUT_UPGRADE_ONLY`.
```

Samples and applications built for Thingy:53 enable simultaneous update of multiple images by default.
To learn more about Thingy:53, see :ref:`ug_thingy53`.

> 

Reviewer: ```suggestion
```

Reviewer: This looks like a typo
